administrator/components/com_admin/src/View/Help/HtmlView.php

```diff
@@ -94,4 +82,21 @@ protected function addToolbar(): void
     {
         ToolbarHelper::title(Text::_('COM_ADMIN_HELP'), 'support help_header');
     }
+
+    /**
+     * Method to render a given level of a menu using provided layout file
+     *
+     * @param   string      $layoutFile  The layout file to be used to render
+     * @param   array       $menu        The menu to render the children of
+     *
+     * @return  void
+     *
+     * @since   3.8.0
```

Comment:
Is this new? If so then please change the @since as below

```suggestion
     * @since   __DEPLOY_VERSION__
```

Reply:
Changed as suggested. I have not met that one before.